components/button/Button.js

```diff
@@ -54,18 +54,17 @@ const ButtonCore = styled.button.attrs(({ active }) => ({ className: active ? 'a
 	}
 
 	> svg {
-		margin-right: ${props => (props.hasChildren ? props.theme.space[2] : '')};
+		margin-right: ${props => props.hasChildren && props.theme.space[2]};
 	}
 
-	${({ isLoading }) =>
-		isLoading
-			? css`
-					color: transparent !important;
-					& > :not(:first-child) {
-						visibility: hidden;
-					}
-			  `
-			: ''}
+	${({ loading }) =>
+		loading &&
```

Comment:
```suggestion
	${({ isLoading }) =>
		isLoading &&
```

See: https://github.com/Faithlife/styled-ui/commit/2a5aa6623062eea7d78b2820246874f648e469ab

Reply:
Whoops, thanks!